DiegoFlores/Day2/Challenge/Main.java

```diff
@@ -0,0 +1,47 @@
+package person;
+
+    	public static final String STUDENT1CAREER = "architecture";
+	public static final String STUDENT1NAME = "John";
+	public static final String STUDENT1SURNAME = "Connor";
+
+	public static final String STUDENT2CAREER = "Biology";
+	public static final String STUDENT2NAME = "Anakin";
+	public static final String STUDENT2SURNAME = "Skywalker";
+
+	public static final String TEACHER1COURSE  = "Structures";
+	public static final String TEACHER1TIME = "Full time";
+	public static final String TEACHER1NAME = "Obi-Wan";
+	public static final String TEACHER1SURNAME = "Kenobi";
+
+
+public class Main {
+
+	public static void main(String[] args) {
+		
+		Student student1 = new Student(000001,"architecture",1);
+			student1.setNames("John");
```

Comment:
What is the reason to not use the final variables that you alredy have? change all